subsys/zigbee/osif/zb_nrf_transceiver.c

```diff
@@ -26,6 +28,9 @@ BUILD_ASSERT(IS_ENABLED(CONFIG_NET_PKT_TIMESTAMP), "Timestamp is required");
 BUILD_ASSERT(!IS_ENABLED(CONFIG_IEEE802154_NET_IF_NO_AUTO_START),
 	     "Option not supported");
 
+/* Required by workaround for KRKNWK-12301. */
```

Comment:
@greg-fer Are we supposed to mention issues (in known issues/release notes) that we have created workarounds for?